src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/TaskLogsTab.php

```diff
@@ -142,6 +145,15 @@ public function render(): void
         $this->renderPagination();
     }
 
+    public function renderBodyDateCreated($value)
+    {
+        $dateCreated = DatabaseConnection::formatFromDatabaseDateIfNotEmpty($value);
+        $dateTime = $dateCreated ? DateTimeHelper::formatForDisplay($dateCreated) : '-';
```

Comment:
no needed if implemeneted in formatForDisplay

Reply:
Returned `-` on formatForDisplay  if null.